File: edu.wpi.first.wpilib.plugins.cpp/resources/templates/examples/PacGoat/src/Commands/Collect.cpp
#include "Collect.h"

#include "Commands/SetCollectionSpeed.h"
#include "Commands/CloseClaw.h"
#include "Commands/SetPivotSetpoint.h"
#include "Commands/WaitForBall.h"
#include "Robot.h"

Collect::Collect() {
	AddSequential(new SetCollectionSpeed(Collector::kForward));
	AddParallel(new CloseClaw());
	AddSequential(new SetPivotSetpoint(Pivot::kCollect));
	AddSequential(new WaitForBall());
}

File: edu.wpi.first.wpilib.plugins.java/resources/templates/examples/PacGoat/src/org/usfirst/frc/team190/pacgoat/commands/Collect.java
package $package.commands;

import edu.wpi.first.wpilibj.command.CommandGroup;

import $package.subsystems.Collector;
import $package.subsystems.Pivot;

/**
 * Get the robot set to collect balls.
 */
public class Collect extends CommandGroup {
	public Collect() {
		addSequential(new SetCollectionSpeed(Collector.kForward));
		addParallel(new CloseClaw());
		addSequential(new SetPivotSetpoint(Pivot.kCollect));
		addSequential(new WaitForBall());
	}
}
